Sources/SymbolKit/SymbolGraph/Symbol/Location.swift

```diff
@@ -26,6 +26,19 @@ extension SymbolGraph.Symbol {
          */
         public var uri: String
 
+        /**
+         The file URL of the source file where the symbol was originally declared.
+         */
+        @available(macOS 10.11, *)
+        public var url: URL? {
+            // The URI string provided in the symbol graph file may be an invalid URL (rdar://69242070)
+            //
+            // Using `URL.init(dataRepresentation:relativeTo:)` here handles URI strings with unescaped
+            // characters without trying to escape or otherwise process the URI string in SymbolKit.
+            // Remove this workaround as part of rdar://77336041 when rdar://69242070 is resolved.
```

Comment:
I think this comment can be removed with the extra tests you added that have pre-encoded strings. It seems like this implementation will work even when the symbol graph starts properly encoding `uri` properties.

Reply:
I prefer to keep the comment, at least for now. It explains why the `dataRepresentation` initializer is used instead of one of the `fileURLWithPath` initializers, which wouldn't work with strings that contains invalid data. 

We may decide to keep using this, at which point we could remove that last "Remove this workaround..." sentence but I think that the the rest of the comment would remain valuable.